cli/command/service/update_test.go

```diff
@@ -808,3 +808,23 @@ func TestUpdateNetworks(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, is.DeepEqual([]swarm.NetworkAttachmentConfig{{Target: "id999"}}, svc.TaskTemplate.Networks))
 }
+
+func TestUpdateMaxReplicas(t *testing.T) {
+	ctx := context.Background()
+
+	svc := swarm.ServiceSpec{
+		TaskTemplate: swarm.TaskSpec{
+			ContainerSpec: &swarm.ContainerSpec{},
+			Placement: &swarm.Placement{
+				MaxReplicas: 1,
+			},
+		},
+	}
+
+	flags := newUpdateCommand(nil).Flags()
+	flags.Set(flagMaxReplicas, "2")
+	err := updateService(ctx, nil, flags, &svc)
+	assert.NilError(t, err)
+
+	assert.Check(t, is.DeepEqual(svc.TaskTemplate.Placement, &swarm.Placement{MaxReplicas: uint64(2)}))
```

Comment:
Also a nit (but thought I'd leave it as a suggestion); `assert.Check()` uses `t.Fail` instead of `t.FailNow`, so that the test doesn't terminate immediately, this allows the test to continue (if further asserts should still be checked for).

Given that this is the last assert, this could be simplified to;

```suggestion
	assert.DeepEqual(t, svc.TaskTemplate.Placement, &swarm.Placement{MaxReplicas: uint64(2)})
```